client/app/components/composites/ListingCardPanel/ListingCardPanel.js

```diff
@@ -0,0 +1,38 @@
+import React, { Children, Component, PropTypes } from 'react';
+import { div } from 'r-dom';
+import classNames from 'classnames';
+import css from './ListingCardPanel.css';
+
+
+class ListingCardPanel extends Component {
+
+  shouldComponentUpdate(nextProps) {
+    return this.props.children !== nextProps.children;
+  }
+
+  render() {
+    const childrenWithColumnStyle = Children.map(
+      this.props.children,
+      (child) => React.cloneElement(child, {
```

Comment:
It seems so. That's due immutable nature of current React:
https://facebook.github.io/react/blog/2015/03/03/react-v0.13-rc2.html

However, I don't think it matters much whether we pass in pure data (props) or React components, since those components are a lightweight versions of DOM elements. 

In this case passing in child components makes more sense than props of child components (since that would mean that the panel needs to create and understand what's inside of it).